frontend/src/app/password-reset/password-reset.component.scss

```diff
@@ -0,0 +1,3 @@
+label {
```

Comment:
It looks so generic, that I believe we could store it in globas styles instead of component-wide one.

Reply:
@magul I was thinking about this line as well. For now, I think it is better if we keep it scoped here. It is actually a better practice than putting a global label {} selector, which would be the worst we could do here, as 'label' selector is to generic - also in rendering phase when browser queries the DOM to apply styles, generic selectors take longer to query.  

If you really want it to be moved, then the selector in global style should look at least like:
`volontulo-password-reset label { //style rules here }`
to keep it scoped in the component.

If we want to have a properly designed css solution for inputs (and any other parts of the app), then we should stop monkey-patching Bootstrap classes and create a newly written and styled css classes for them, where we can use Bootstrap mixins as well. This would allow us to maintain consistent styling accross the app.